hpa/sf-perftest/README.md

```diff
@@ -23,7 +23,7 @@ Replace the value of the `TEST_CASE` env variable with your test case e.g.,`my-o
 ## Create an experiment
 
 Create the RBAC permission
-`redskyctl generate rbac -f sf-experiment/experiment.yaml | kubectl apply -f -`
+`stormforge generate rbac -f sf-experiment/experiment.yaml | kubectl apply -f -`
```

Comment:
This one is the other sf-test based experiment. Let me know if you would like me to drop any hints in the README about having `forge` setup. This one is a bit different in that there is no `app.yaml`, and the `experiment.yaml` already exists.

Reply:
Yes, `stormforge generate rbac` is fine, we aren't getting rid of all the generate subcommands, just `app` and `exp`.

Starting with 2.0 the goal is not to require `forge` for new customers, instead allowing them to do everything through the UI. Since Optimize authorization tokens can now be accepted by Performance (via the exchange), we no longer require `forge` (you would still need it if, for example, you wanted to create your test cases via the CLI).